src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -421,17 +406,15 @@ public synchronized String getId() {
         return metaDaten[ListeFilme.FILMLISTE_ID_NR];
     }
 
-    public synchronized String genDateRev() {
-        // Tag, Zeit in lokaler Zeit wann die Filmliste erstellt wurde
-        // in der Form "yyyy.MM.dd__HH:mm"
+/*    public synchronized String genDateRev() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)